rules.md

```diff
@@ -5,7 +5,7 @@
 <!-- core -->
 | Lint Rules | Description | [Fix][] |
 | :--------- | :---------- | ------- |
-| [`avoid_empty_else`](https://dart.dev/lints/avoid_empty_else) | Avoid empty else statements. | ✅ |
+| [`avoid_empty_else`](https://dart.dev/lints/avoid_empty_else) | Avoid empty statements in else clauses. | ✅ |
```

Comment:
in -> as

(Does the lint only target `;`, or also `{}`?)

Reply:
I believe the ultimate system-of-record for this info are the lint descriptions at https://github.com/dart-lang/sdk/tree/main/pkg/linter/lib/src/rules. Any changes there will flow here (in a few steps).